app/views/users/show.html.erb

```diff
@@ -1 +1,33 @@
-<h1>The page of the user with id:</h1>
\ No newline at end of file
+<main class='user-details'>
+  <section class='user-header'>
+    <div class='user-photo'>
+      <img src= <%= @user.photo %> class='user-image'>
+    </div>
+
+    <div class='user-info'>
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </section>
+
+  <section class='user-card'>
+    <div class='bio'>
+      <h3>Bio</h3>
+      <p><%= @user.bio %></p>
+    </div>
+
+    <% if @user.last_posts.blank? %>
+      <h2>There are no posts. Please add a post.</h2>
+    <% else %>
+      <% @user.last_posts.each do |post| %>
+      <div class='post-container'>
+        <h3>Post: # <%= link_to post.id, user_post_path(@user, post) %></h3>
+        <p><%= post.text %></p>
+        <p class='counter'>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+      </div>
+      <% end %>
+    <% end %>
+  </section>
+
```

Comment:
- [ ] it would be better to add spacing margin and styles to put the elements as in the wireframe 
- the profile picture is bigger than expected and not well positioned as the wireframe 
-  all the see all posts button should be centered 
- all that so that you match the wireframe 
- Yours 

<img width="897" alt="Screenshot 2023-06-02 at 11 34 28" src="https://github.com/lugard1/blogapp/assets/61790587/da62c087-b357-4170-8e1d-c458565d7305">



- Design 

<img width="455" alt="Screenshot 2023-06-02 at 11 26 44" src="https://github.com/lugard1/blogapp/assets/61790587/bf94a21e-9ac1-4779-8128-e3a380598e91">